Frontend/CO.CDP.OrganisationApp/Constants/OrganisationSchemeType.cs

```diff
@@ -1,26 +1,46 @@
+using System.Collections.Generic;
+
 namespace CO.CDP.OrganisationApp.Constants;
 
 public static class OrganisationSchemeType
 {
-    public static Dictionary<string, string> OrganisationScheme => new()
-        {
-            { "GB-COH", "Companies House Number"},
-            { "GB-CHC", "Charity Commission for England & Wales Number"},
-            { "GB-SC", "Scottish Charity Regulator"},
-            { "GB-NIC", "Charity Commission for Northern Ireland Number"},
-            { "GB-MPR", "Mutuals Public Register Number"},
-            { "GG-RCE", "Guernsey Registry Number"},
-            { "JE-FSC", "Jersey Financial Services Commission Registry Number"},
-            { "IM-CR", "Isle of Man Companies Registry Number"},
-            { "GB-NHS", "National health Service Organisations Registry Number"},
-            { "GB-UKPRN", "UK Register of Learning Provider Number"},
-            { "VAT", "VAT number"},
-            { "Other", "Other / None"},
-            { "GB-PPON", "Ppon" }
-        };
+    public const string CompaniesHouseNumber = "GB-COH";
```

Comment:
Naming is t consistent.  “Number” is only used in two cases. Can we drop it to keep scheme naming consistent?

Reply:
Done!! renamed to CompanyHouse

Reply:
VATNumber suffers the same inconsistency